rr_evgp/CMakeLists.txt

```diff
@@ -6,29 +6,27 @@ set(CMAKE_CXX_STANDARD 17)
 find_package(catkin REQUIRED COMPONENTS
         roscpp
         rospy
-        rr_common
-        rr_description
-        rr_gazebo
-        rr_platform
+        rr_msgs
         sensor_msgs
         geometry_msgs
         costmap_2d
         parameter_assertions
 )
 
+find_package(OpenCV 3 REQUIRED)
 
 catkin_package(
 #  INCLUDE_DIRS include
 #  LIBRARIES rr_evgp
 #  CATKIN_DEPENDS geometry_msgs roscpp rospy rr_common rr_description rr_gazebo rr_platform sensor_msgs
-#  DEPENDS system_lib
+  DEPENDS OpenCV
 )
 
 
 include_directories(
 # include
   ${catkin_INCLUDE_DIRS}
+  ${OpenCV_INCLUDE_DIRS}
 )
 
-add_library(rr_evgp_plugins src/binary_bayes_filter_obstacle_layer.cpp)
-target_link_libraries(rr_evgp_plugins ${catkin_LIBRARIES})
```

Comment:
Why were these removed?

Reply:
Pretty sure I moved them to another cmakelists file inside src